STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -416,9 +415,9 @@ the renewal timers needed by the NDC to inform its certificate reload logic.
 ~~~
 
 If an `identifier` attribute of type `dns` was included, the IdO MUST validate
-the specified CNAME at this point in the flow.  At the minimum, the IdO MUST
-verify that both DNS names are syntactically valid, to prevent a malicious NDC
-from injecting arbitrary data into a DNS zone file.
+the `cname-map` entry in the configuration at this point in the flow.  At the
```

Comment:
Wait. Now the CNAME mapping is coming from the IdO, so there's nothing to verify here. Of course the IdO needs to verify that the NDC only uses a delegation that it is allowed to (but I think we already say so).

Reply:
good catch!